aqua-ethos-setup/aqua_ethos.sh

```diff
@@ -123,6 +151,90 @@ function makePut {
 	fi
 }
 
+function get_gateway_id {
+    GATEWAY=$(nakeGET servers body| jq "[.[].id]")
+}
+
+function create_label_if_does_not_exist {
+    CREATE_LABEL=$1
+    ENCODED_LABEL=${CREATE_LABEL// /%20}
+    JQ_LABEL_FILTER='.[] | select([.name=="'$CREATE_LABEL'"] | any) | .name'
+    RESPONSE_LABEL=$(makeGet settings/labels body  | jq "$JQ_LABEL_FILTER")
+    if [ -n "$RESPONSE_LABEL" ]; then
+        log "Label \"$CREATE_LABEL\" already exists."
+    else
+        log "Label \"$CREATE_LABEL\" does not exist, creating it."
+        NEW_LABEL=$(makePost settings/labels "{ \"name\": \"$CREATE_LABEL\" }" | jq ".name")
+        if [ "$NEW_LABEL"="$CREATE_LABEL" ]; then
```

Comment:
`makePost` will not return the body of the POST, so I'm not sure what's being compared here